packages/babel-standalone/src/preset-stage-1.ts

```diff
@@ -7,8 +7,6 @@ export default (_: any, opts: any = {}) => {
     useBuiltIns = false,
     decoratorsLegacy = false,
     decoratorsBeforeExport,
-    pipelineProposal = "minimal",
-    pipelineTopicToken = "%",
     recordAndTupleSyntax,
```

Comment:
We should keep these options here (without the default values), and forward them to the Stage 2 preset at lines 18-22.

Reply:
Ok, will update these 2 things later tonight. Tried to do it all inside GH's web editor and it's too difficult. But gotta get back to work now.

Reply:
I'm confused, do you mean the options should stay here and be imported by stage 2? And that the plugin lines below should remain here but not pass options, and be copied into stage 2 with the imported options? That's what it sounds like you're saying but it doesn't make a lot of sense to me to have no options for it here.

Reply:
It should do the same as it does, for example, for `decoratorsBeforeExport`: it accepts the option (without setting a default value), and passes it as-is to the Stage 2 preset below (which then sets the default value).